http-client/http-client.cabal

```diff
@@ -77,6 +77,7 @@ test-suite spec
   hs-source-dirs:      test
   default-language:    Haskell2010
   other-modules:       Network.HTTP.ClientSpec
+  build-tools:         hspec-discover
```

Comment:
What's the motivation for adding this? The builds have been passing with both `cabal-install` and `Stack` without them. I _think_ this is for `new-build`. If so, please add a comment explaining this motivation, otherwise it's likely that future changes may remove it. It would also be a good idea to add a new Travis stanza which includes `new-build` to prevent a change accidentally breaking `new-build` in the future.

Reply:
It's documented [here](http://cabal.readthedocs.io/en/latest/developing-packages.html#pkg-field-build-tools)